### PR TITLE
[Feature] Displaying plan alert (advance settings)

### PR DESCRIPTION
**Pull request by Civolilah**
Here are screenshots of implemented solution for displaying plan warning alert for advance settings:

Screenshot 2022-12-25 at 17 32 17

Screenshot 2022-12-25 at 17 42 15

Screenshot 2022-12-25 at 17 47 47

@beganovich @turbo124 Instead of completely removing advance settings from the sidebar for user who does not have permission for it, we now just disable certain buttons and display a warning message. All advance settings that require a `Pro` or `Enterprise` plan except the last two (Credit Cards & Banks and User Management) where `Enterprise` is required.

- For Credit Cards & Banks is used `upgrade_to_connect_bank_account` translation keyword (Upgrade to Enterprise to connect your bank account)

- For User Management on create page is used `add_users_not_supported` translation keyword (Upgrade to the Enterprise plan to add additional users to your account.)

- All other advance settings pages use `start_free_trial_message` translation keyword (Start your FREE 14 day trial of the pro plan)

### CODE REVIEW

**Comment by turbo124** [comment]
@Civolilah @beganovich LGTM

File: src/pages/settings/client-portal/ClientPortal.tsx [hunk 1]
```diff
@@ -34,16 +39,22 @@ export function ClientPortal() {
...
 
   return (
     <Settings
       title={t('client_portal')}
-      breadcrumbs={pages}
       docsLink="docs/advanced-settings/#client_portal"
+      breadcrumbs={pages}
       onSaveClick={onSave}
       onCancelClick={onCancel}
+      disableSaveButton={!proPlan() && !enterprisePlan() && isHosted()}
```
Comment (beganovich) [inline src/pages/settings/client-portal/ClientPortal.tsx:52]: Do we use this all the time? Can we extract it into hook?

File: src/pages/settings/users/create/Create.tsx [hunk 2]
```diff
@@ -88,11 +90,12 @@ export function Create() {
...
-
-        error.response?.status === 412
-          ? toast.error(t('password_error_incorrect'), { id: toastId })
-          : toast.error(t('error_title'), { id: toastId });
+        if (error.response?.status === 412) {
+          toast.error('password_error_incorrect');
+        } else {
+          toast.error();
+          console.error(error);
+        }
```
Comment (beganovich) [inline src/pages/settings/users/create/Create.tsx:98]: Why?

**Review by beganovich — CHANGES_REQUESTED** [review]
Looks good, small changes are needed.

Most importantly, possible centralization of `!proPlan() && !enterprisePlan() && isHosted()` into hook.

File: src/pages/settings/client-portal/ClientPortal.tsx (hunk 1 above)
Comment (Civolilah) [reply src/pages/settings/client-portal/ClientPortal.tsx:52]: It's extracted to hook and applyed to all places where we use this condition for advance settings.

**Review by beganovich — APPROVED** [review]
Looks awesome, thanks :+1:

**Pull request merged**